pims/bioformats.py

```diff
@@ -93,7 +94,7 @@ def _download_jar(version='5.7.0'):
 
 def _maybe_tostring(field):
     if hasattr(field, 'toString'):
```

Comment:
It seems like `toString()` is a common method of many Java classes. Since it's part of the Java standard library I think it can't be removed from Jpype; rather, it now returns a Java string instead of a Python string, for performance reasons. There's a bit more detail in [this doc](https://jpype.readthedocs.io/en/latest/ChangeLog-0.7.html).

That said, `str(java_obj)` will now call `str(java_obj.toString())` where appropriate. I think that explains the "deprecation" of `toString()`.

Reply:
That's good to know